core_impl/src/test/java/io/opencensus/implcore/stats/IntervalBucketTest.java

```diff
@@ -49,28 +43,24 @@
   private static final Duration MINUTE = Duration.create(60, 0);
   private static final Duration NEGATIVE_TEN_SEC = Duration.create(-10, 0);
   private static final Timestamp START = Timestamp.create(60, 0);
-  private static final BucketBoundaries BUCKET_BOUNDARIES =
-      BucketBoundaries.create(Arrays.asList(-10.0, 0.0, 10.0));
-  private static final List<Aggregation> AGGREGATIONS_V1 =
-      Collections.unmodifiableList(Arrays.asList(
-          Sum.create(), Count.create(), Histogram.create(BUCKET_BOUNDARIES)));
+  private static final Mean MEAN = UnreleasedApiAccessor.createMean();
```

Comment:
createMean should be public now

Reply:
Correct, Mean is exposed in the next PR: https://github.com/census-instrumentation/opencensus-java/pull/630/files#diff-d4eaa57f3d0967756e7c291e24c53968R125, along with other changes on AggregationData.